sse2neon.h

```diff
@@ -6979,12 +6979,12 @@ FORCE_INLINE __m128i _mm_cvtepu32_epi64(__m128i a)
         vmovl_u32(vget_low_u32(vreinterpretq_u32_m128i(a))));
 }
 
-// Converts the four unsigned 8-bit integers in the lower 16 bits to four
-// unsigned 32-bit integers.
+// Converts the eight unsigned 8-bit integers in the lower 64 bits to eight
+// unsigned 16-bit integers.
```

Comment:
Could you replace the comment with the following text?
```c
// Zero extend packed unsigned 8-bit integers in a to packed 16-bit integers,
// and store the results in dst.
// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_cvtepu8_epi16
```

In the future, we would unify the comment of each intrinsic with the corresponding description on [Intel Intrinsic Guide](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#).

Reply:
done